packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -32,10 +32,13 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
   public static defaultProps: INavProps = {
     groups: null
   };
+  private _linkSelectedAriaLabel: string | undefined;
```

Comment:
I don’t think that an aria prop needs an instance property of its own. Enough to just add the prop to the props object destructuring on the line 80